.gitignore

```diff
@@ -28,7 +28,7 @@ buildNumber.properties
 
 #Mono Project Files
 *.pidb
-*.resources
+hedera-node/hedera-mono-service/src/test/resources/
```

Comment:
Why are we excluding the mono svc resources path and removing the `*.resources` file exclusions. This line has nothing to do with the `hedera-mono-service` module and is related to the MonoDevelop IDE exclusions. 

These changes should be reverted.

Reply:
Reverted now 👍 Thanks @nathanklick !